### PR TITLE
feat: consistently prefix wash-cli arguments

### PR DESCRIPTION
**Pull request by hailelagi**
## Feature or Problem

consistent namespace prefixes for cli flags

## Related Issues

https://github.com/wasmCloud/wasmCloud/issues/1102

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -190,14 +201,16 @@ struct Args {
     #[clap(
         long = "oci-user",
         env = "OCI_REGISTRY_USER",
+        alias = "WASMCLOUD_OCI_REGISTRY_USER",
```
Comment (vados-cosmonic) [inline src/main.rs:204]: Any reason you didn't go for `WASMCLOUD_*` being the `env` value here like other places?

**Review by vados-cosmonic — APPROVED** [review]
LGTM :rocket: 

Thanks for the contribution @hailelagi, let's run the CI and we should be good to merge.

File: src/main.rs (hunk 1 above)
Comment (hailelagi) [reply src/main.rs:204]: good catch! not in particular, glossed over this thanks

**Review by brooksmtownsend — APPROVED** [review]
Awesome! One more task checked off the 1.0 list, thank you @hailelagi

**Pull request merged**